README.md

```diff
@@ -1,56 +1,121 @@
-[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](CONTRIBUTING.md)
 [![Maven Central](https://img.shields.io/maven-central/v/io.openliberty/openliberty-runtime.svg?label=Maven%20Central)](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22io.openliberty%22%20a%3A%22openliberty-runtime%22)
+[![Docker Pulls](https://img.shields.io/docker/pulls/_/open-liberty.svg?color=yellow)](https://hub.docker.com/_/open-liberty)
+[![Website](https://img.shields.io/badge/website-live-purple.svg)](https://openliberty.io/)
+[![Stack Overflow](https://img.shields.io/badge/find-answers-blue.svg)](https://stackoverflow.com/questions/tagged/open-liberty)
+[![Groups.io](https://img.shields.io/badge/ask-groups.io-orange.svg)](https://groups.io/g/openliberty)
+[![License](https://img.shields.io/badge/License-EPL%201.0-green.svg)](https://opensource.org/licenses/EPL-1.0)
+[![Twitter](https://img.shields.io/twitter/follow/openlibertyio.svg?style=social&label=Follow)](https://twitter.com/OpenLibertyIO)
 
 # Summary
-Open Liberty is a highly composable, fast to start, dynamic application server runtime environment.
-
-# Table of Contents
-* [Getting Started](https://github.com/OpenLiberty/open-liberty#getting-started)
-* [Contribute to Open Liberty](https://github.com/OpenLiberty/open-liberty#contribute-to-open-liberty)
-* [Community](https://github.com/OpenLiberty/open-liberty#community)
+A lightweight open framework for building fast and efficient cloud-native Java microservices:
+* Open Liberty is fast to start up with low memory footprint and live reload for quick iteration.
+* Simple to add and remove features from the latest versions of MicroProfile and Java EE.
+* Zero migration lets you focus on what's important, not the APIs changing under you.
 
 ## Getting Started
 
-### Downloading
-You can download released versions and nightly build artifacts from: https://www.openliberty.io/downloads/
-
-### Quick start
+### Launch a sample app using [Maven](https://maven.apache.org/)
     git clone https://github.com/OpenLiberty/sample-getting-started.git
     cd sample-getting-started
     mvn clean package liberty:run-server
 
 Open browser to http://localhost:9080/ and explore the demo application.
 
-### Open Liberty Guides
-Visit the [OpenLiberty website](https://openliberty.io/guides/) for a number of step by step guides.
+See [this guide](https://openliberty.io/guides/getting-started.html)  to learn how it works.
+
+### Downloads
+
+* [Maven Central](https://mvnrepository.com/artifact/io.openliberty/openliberty-runtime)
+* [Open Liberty website](https://www.openliberty.io/downloads/)
+* [Docker Hub](https://hub.docker.com/_/open-liberty)
+
+## Usage
+
+
+### Docker
+
+* [This guide](https://openliberty.io/guides/docker.html) demonstrates development using Docker containers. 
+
+### Maven
+
+* [This guide](https://openliberty.io/guides/maven-intro.html) demonstrates development using the Liberty Maven Plugin. 
 
+### Command Line
```

Comment:
Similar to how we link off to external resources for Docker/Maven, lets link off to the OpenLiberty.io doc page on using the server commands:
https://openliberty.io/docs/ref/command/

Reply:
ok fair enough - making change